src/index.html

```diff
@@ -3,7 +3,7 @@
     <head>
         <meta charset="utf-8">
         <meta http-equiv="x-ua-compatible" content="ie=edge">
-        <title></title>
+        <title>Etsy Listing Page</title>
```

Comment:
Please move this code into `product.html`... Maybe coordinate with @jeremyrist on renaming that to `listing.html` as well.